deploy/gk-deploy

```diff
@@ -515,14 +508,37 @@ if [[ "${EXECUTOR}" == "ssh" ]]; then
   done <<< "$(echo -e "${NODES}")"
 fi
 
-if [[ ${LOAD} -eq 0 ]]; then
-  output -n "Checking that heketi pod is not running ... "
-  check_pods "glusterfs=heketi-pod" 2
-  if [[ $? -eq 0 ]]; then
-    output "Found heketi pod running. Please destroy existing setup and try again."
-    exit 1
-  fi
-  output "OK"
+output "Checking for pre-existing resources..."
+
+output -n "  GlusterFS pods ... "
+check_pods "glusterfs=pod" 2 2>&1
+if [[ $? -eq 0 ]]; then
+  EXISTS_GLUSTERFS=1
+  output "found."
+else
+  output "not found."
+fi
+
+output -n "  deploy-heketi pod ... "
+check_pods "deploy-heketi=pod" 2 2>&1
+if [[ $? -eq 0 ]]; then
+  EXISTS_DEPLOY_HEKETI=1
+  output "found."
+else
+  output "not found."
+fi
+
+output -n "  heketi pod ... "
+check_pods "heketi=pod" 2 2>&1
+if [[ $? -eq 0 ]]; then
+  EXISTS_HEKETI=1
+  output "found."
+else
+  output "not found."
+fi
+
+if [[ ${EXISTS_HEKETI} -eq 0 ]]; then
+  output -n "Creating initial resources ... "
   if [[ "${CLI}" == *oc\ * ]]; then
     eval_output "${CLI} create -f ${TEMPLATES}/deploy-heketi-template.yaml 2>&1"
```

Comment:
hmm, why would we want to always create the deploy-heketi thing when heketi does not exist? maybe we already created deploy-heketi?